(MainThread) 221105_21:29:42.241 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1210 now is 2022-11-05 21:29:42.241369
(MainThread) 221105_21:29:43.455 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1267 serverVersion:157 connectionTime:b'20221105 21:29:43 CET'
(MainThread) 221105_21:29:43.455 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1273 Avant creation thread...
(MainThread) 221105_21:29:43.456 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1275 Après creation thread...
(MainThread) 221105_21:29:43.457 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1277 Après start thread...
(MainThread) 221105_21:29:43.459 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1280 Dans start...
(MainThread) 221105_21:29:43.461 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1354 Appel requete reqHistoricalData 15 mins...NASDAQ-miniIdReq=2
(MainThread) 221105_21:29:43.463 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1365 Appel requete reqHistoricalData 5 mins...NASDAQ-miniIdReq=3
(Thread-2 (run_loop)) 221105_21:29:43.466 ERROR wrapper.py:56 ERROR 2 2174 Avertissement: You submitted request with date-time attributes without explicit time zone. Please switch to use yyyymmdd-hh:mm:ss in UTC or use instrument time zone, like US/Eastern. Implied time zone functionality will be removed in the next API release
(MainThread) 221105_21:29:43.466 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1375 Appel requete reqHistoricalData 1 min...NASDAQ-miniIdReq=4
(Thread-2 (run_loop)) 221105_21:29:43.467 ERROR wrapper.py:56 ERROR -1 2104 La connexion de données de marché est OK:eufarm
(Thread-2 (run_loop)) 221105_21:29:43.469 ERROR wrapper.py:56 ERROR -1 2104 La connexion de données de marché est OK:eufarmnj
(MainThread) 221105_21:29:43.469 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1354 Appel requete reqHistoricalData 15 mins...NASDAQ-miniIdReq=6
(Thread-2 (run_loop)) 221105_21:29:43.470 ERROR wrapper.py:56 ERROR -1 2106 La connexion de données HMDS est OK:euhmds
(MainThread) 221105_21:29:43.472 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1365 Appel requete reqHistoricalData 5 mins...NASDAQ-miniIdReq=7
(Thread-2 (run_loop)) 221105_21:29:43.472 ERROR wrapper.py:56 ERROR -1 2106 La connexion de données HMDS est OK:fundfarm
(MainThread) 221105_21:29:43.475 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1375 Appel requete reqHistoricalData 1 min...NASDAQ-miniIdReq=8
(Thread-2 (run_loop)) 221105_21:29:43.475 ERROR wrapper.py:56 ERROR -1 2106 La connexion de données HMDS est OK:ushmds
(Thread-2 (run_loop)) 221105_21:29:43.478 ERROR wrapper.py:56 ERROR -1 2158 La connexion de la batterie de serveurs Sec-def est OK:secdefeu
(MainThread) 221105_21:29:43.478 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1354 Appel requete reqHistoricalData 15 mins...DOW-miniIdReq=10
(MainThread) 221105_21:29:43.481 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1365 Appel requete reqHistoricalData 5 mins...DOW-miniIdReq=11
(MainThread) 221105_21:29:43.492 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1375 Appel requete reqHistoricalData 1 min...DOW-miniIdReq=12
(MainThread) 221105_21:29:43.494 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1354 Appel requete reqHistoricalData 15 mins...DOW-miniIdReq=14
(MainThread) 221105_21:29:43.497 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1365 Appel requete reqHistoricalData 5 mins...DOW-miniIdReq=15
(MainThread) 221105_21:29:43.504 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1375 Appel requete reqHistoricalData 1 min...DOW-miniIdReq=16
(MainThread) 221105_21:29:43.507 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1354 Appel requete reqHistoricalData 15 mins...DAX-miniIdReq=18
(MainThread) 221105_21:29:43.510 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1365 Appel requete reqHistoricalData 5 mins...DAX-miniIdReq=19
(MainThread) 221105_21:29:43.512 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1375 Appel requete reqHistoricalData 1 min...DAX-miniIdReq=20
(MainThread) 221105_21:29:43.514 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1354 Appel requete reqHistoricalData 15 mins...DAX-miniIdReq=22
(MainThread) 221105_21:29:43.517 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1365 Appel requete reqHistoricalData 5 mins...DAX-miniIdReq=23
(MainThread) 221105_21:29:43.519 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1375 Appel requete reqHistoricalData 1 min...DAX-miniIdReq=24
(MainThread) 221105_21:29:43.522 ERROR GT_01_GetHLCS_15m_5m_1m_v2.py:1397 Après lancement thread ib...
